src/pytorch_lightning/strategies/fully_sharded.py

```diff
@@ -190,35 +248,29 @@ def configure_ddp(self) -> None:
             # (TODO: need to figure out solution)
             self.model_to_device()
```

Comment:
this is incorrect too. We need to move the model to the device only in case of `ssd-offload`. I will do it in a follow-up.